src/Schema/TypeRegistry.php

```diff
@@ -41,7 +41,7 @@ class TypeRegistry
      *
      * @var array<string, \GraphQL\Type\Definition\Type>
      */
-    protected $types;
+    protected $types = [];
```

Comment:
FYI: I had to add a default value, because when I tried to call `$typesRegistry->possibleTypes()` without to register any type at all, it was throwing an error, because `possibleTypes()` expects to return an array, but null was returned, because `$types` were not defined and not manipulated.

Reply:
Good idea regardless.